invenio_formatter/templates/semantic-ui/invenio_formatter/macros/badges.html

```diff
@@ -9,37 +9,44 @@
 
 {# Generate a list different ways to embed a badge. #}
 {%- macro badges_formats_list(image_filename, target, alternative='DOI') %}
+
+  <style>
+    .ui.message {
+      overflow: auto;
+    }
+  </style>
```

Comment:
Is this needed?

Reply:
Yes, because before it got out of the modal bounds.

Reply:
True, sorry, I missed that